recipes/pairix/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: pairix
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/4dn-dcic/pairix/archive/{{ version }}.tar.gz
+  md5: 05ac0be28a85e4d6bdca744ddec8824e
+
+build:
+  number: 0
+  
+requirements:
+  build:
+    - gcc
+    - zlib
+    - python
+    - setuptools
+
+  run:
+    - python
+    - htslib
+
+test:
+  imports:
+    - pypairix
+  commands:
+    - which pairix
```

Comment:
I couldn't find a benign option that returns an exit code of 0. @SooLee, does one exist or would it be possible to add a `--help` option that exits with 0?

Originally I tried to pull in sample files and run a test using:
```
  source_files:
    - samples/test_4dn.pairs.gz
    - samples/test_4dn.pairs.gz.px2
  commands:
    - pairix -l samples/test_4dn.pairs.gz
```
but never got it to work. The build system couldn't find the source files.